features/steps/cpts_ui/prescription_list_steps.py

```diff
@@ -0,0 +1,110 @@
+from behave import given, when, then  # pyright: ignore [reportAttributeAccessIssue]
+from playwright.sync_api import expect
+
+from pages.prescription_list_page import PrescriptionListPage
+
+
+@when('I search for a prescription using a valid prescription ID "{prescription_id}"')
+def search_using_prescription_id(context, prescription_id):
+    # Simply use the Find a prescription button directly, skip the tab click since it's causing issues
+    context.page.get_by_role("button", name="Find a prescription").click()
```

Comment:
This should use a `data-testid` tag, rather than looking for text. It's a language thing - we might want to test in Welsh.